### ISSUE TITLE
Add instructions to find upstream in README

### ISSUE THREAD

**Issue by matthewlmcclure** [emacsmirror/p.old#27]
I have a change to sr-speedbar that I'd like to contribute. I found your mirror, but I can't tell how to find the upstream. In this particular case, I see there's a reference to the Emacs wiki in the sr-speedbar source code, so I assume that's the relevant upstream, but it would be nice to have some info about how to find the upstream that emacsmirror has configured for an arbitrary repository.

**Comment by tarsius** [comment]
Yes, I am mirroring that package from the Emacswiki. It even has a page there: http://emacswiki.org/SrSpeedbar.

Unfortunately updating packages from the Emacswiki is currently broken, see #26. There are a few other issues and once they are resolved I would also like to provide a little mode that shows information about mirrored packages similar to how `package.el` does it.

But I got sidetracked maintaining Magit... I am currently working on finishing up for the next Magit release and once that is done, I will get back to making the necessary adjustments to the Emacsmirror. So yes, I will implement these things, but I won't be able to work on it for at least the next two months.

**Comment by tarsius** [comment]
Once wiki import works again the url extracted from the library should automatically be used as the "homepage" link on https://github.com/emacsmirror/sr-speedbar.
